providers/catalog.go

```diff
@@ -8,6 +8,7 @@ const (
 	Salesforce Provider = "salesforce"
 	Hubspot    Provider = "hubspot"
 	LinkedIn   Provider = "linkedIn"
+	SalesLoft  Provider = "salesLoft"
```

Comment:
Can we name this as `Salesloft`? From their website, seems like they refer to themselves as a single word

Reply:
Approved - feel free to 'Squash and merge' once you resolve this.